README.md

```diff
@@ -32,6 +32,8 @@ Bad English &lt;-> Spanish translations.
 - They who don't cry, aren't breastfed.
 - It licks me an egg.
 - Pulled from the hairs
+- You touched dance to the ugliest
```

Comment:
"It touched you" goes out with fries.

I would even say "you're dancing with the ugliest". I try to picture the sentences being used in a quick informal conversation with fellow spanglishers.